DTS-STN/Alpha-Site
# Contact Us link

# Description

[Update Contact Us link](https://trello.com/c/C9DmZyej)

Updated the Contact Us link in the footer to link to a Contact Us section on the about page. Added a header on the about page to show the section better.

While I was updating the footer, I noticed the rest of the footer links didn't link to their respective french pages while navigating the site in french, so I fixed that too.

## Acceptance Criteria

The Contact Us link in the footer now links to the Contact Us section on the about page.

## Test Instructions

1. Navigate to the home page
2. Scroll to the footer and click on Contact Us
3. See that it navigates to `/about#contact-us`


## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
